### ISSUE TITLE
Syntax error in file default-init.in

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/nagioscore#558]
There is a syntax error in the default-init.in file in line 70:
https://github.com/NagiosEnterprises/nagioscore/blob/a70955475673fe1e3a5d5adf94e62f4b2c8b3ca5/startup/default-init.in#L70

It should be:
```bash
	mount -t tmpfs -o uid=${NagiosUserUID},gid=${NagiosGroupGID},size=${RAMDISK_SIZE}m tmpfs ${RAMDISK_DIR}
```

If the auto create of a ramdisk is active, this syntax error results in:

```bash
21:37:39 [root@nagios4]:/usr/local/src/nagioscore# service nagios restart
+ test -f /etc/sysconfig/nagios
+ . /etc/sysconfig/nagios
++ USE_RAMDISK=1
++ RAMDISK_SIZE=200
++ RAMDISK_DIR=/tmp/test_nagios_ramdisk
+ USE_RAMDISK=1
+ test 1 -ne 0
+ test 200X '!=' X
++ grep '/tmp/test_nagios_ramdisk type tmpfs'
++ mount
+ ramdisk=
+ '[' X == X ']'
+ '[' '!' -d /tmp/test_nagios_ramdisk ']'
+ mkdir -p -m 0755 /tmp/test_nagios_ramdisk
+ '[' -d /tmp/test_nagios_ramdisk ']'
++ id -u nagios
+ NagiosUserUID=501
++ id -g nagios
+ NagiosGroupGID=501
+ mount -t tmpfs -o 'uid=501,gid={501},size=200m' tmpfs /tmp/test_nagios_ramdisk
mount: wrong fs type, bad option, bad superblock on tmpfs,
       missing codepage or helper program, or other error
       (for several filesystems (e.g. nfs, cifs) you might
       need a /sbin/mount.<type> helper program)
       In some cases useful info is found in syslog - try
       dmesg | tail  or so

+ su nagios -c 'mkdir -m 0755 /tmp/test_nagios_ramdisk/checkresults'
mkdir: cannot create directory `/tmp/test_nagios_ramdisk/checkresults': Permission denied
+ set +x
Running configuration check... + test -f /etc/sysconfig/nagios
+ . /etc/sysconfig/nagios
++ USE_RAMDISK=1
++ RAMDISK_SIZE=200
++ RAMDISK_DIR=/tmp/test_nagios_ramdisk
+ USE_RAMDISK=1
+ test 1 -ne 0
+ test 200X '!=' X
++ grep '/tmp/test_nagios_ramdisk type tmpfs'
++ mount
+ ramdisk=
+ '[' X == X ']'
+ '[' '!' -d /tmp/test_nagios_ramdisk ']'
+ set +x
Stopping nagios: .^[[A^[[Adone.
+ test -f /etc/sysconfig/nagios
+ . /etc/sysconfig/nagios
++ USE_RAMDISK=1
++ RAMDISK_SIZE=200
++ RAMDISK_DIR=/tmp/test_nagios_ramdisk
+ USE_RAMDISK=1
+ test 1 -ne 0
+ test 200X '!=' X
++ grep '/tmp/test_nagios_ramdisk type tmpfs'
++ mount
+ ramdisk=
+ '[' X == X ']'
+ '[' '!' -d /tmp/test_nagios_ramdisk ']'
+ set +x
Starting nagios: Running configuration check... done.
21:37:41 [root@nagios4]:/usr/local/src/nagioscore#
```

**Comment by ccztux** [comment]
If the auto create of a ramdisk is active, this syntax error results in:

```bash
21:37:57 [root@nagios4]:/usr/local/src/nagioscore# service nagios restart
mount: wrong fs type, bad option, bad superblock on tmpfs,
       missing codepage or helper program, or other error
       (for several filesystems (e.g. nfs, cifs) you might
       need a /sbin/mount.<type> helper program)
       In some cases useful info is found in syslog - try
       dmesg | tail  or so

mkdir: cannot create directory `/tmp/test_nagios_ramdisk/checkresults': Permission denied
Running configuration check... Stopping nagios: done.
Starting nagios: Running configuration check... done.
21:38:00 [root@nagios4]:/usr/local/src/nagioscore# 
```

**Comment by jomann09** [comment]
Merged this in for 4.4.2
